doc/ale-sh.txt

```diff
@@ -2,6 +2,25 @@
 ALE Shell Integration                                          *ale-sh-options*
 
 
+===============================================================================
+sh-language-server                                     *ale-sh-language-server*
+
+g:ale_sh_language_server_executable        *g:ale_sh_language_server_executable*
+                                          *b:ale_sh_language_server_executable*
```

Comment:
Make sure to align tags to the right margin.

Reply:
done